src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: content-box;
```

Comment:
`content-box` is a default value, maybe you wanted `border-box` ?
and use `box-sizing` for all elements `* {...}`

Reply:
This task was realy hard for me. I tried everything. )